### PR TITLE
Upgraded Sidebar Presence

### PR DESCRIPTION
**Pull request by danny98m**
## Sidebar Presence Now Made Known Through Popover
The first time a user uses Lexos a popover will appear alerting the user of the in the margins tool. I'm using local web storage so this will only happen once unless the user clears their browser's cache. 
Here is a terrible quality gif:
sidebar

### CODE REVIEW

**Comment by danny98m** [comment]
Here is the relevant issue [#696](https://github.com/WheatonCS/Lexos/issues/696).

**Comment by scottkleinman** [comment]
The popover could use some styling (maybe wider), but otherwise I think this implementation is good. You could possibly reset local storage to remove 'visited' when the session is restarted, but I am undecided as to whether that's a good idea.

**Comment by danny98m** [comment]
I added some style to this popover. If you would prefer session storage rather than local storage let me know and I'll change it.

**Comment by mleblanc321** [comment]
@danny98m - give me a demo of this and we'll discuss

File: lexos/static/js/scripts_base.js [hunk 1]
```diff
@@ -1,3 +1,23 @@
+$(function () {
+  if (localStorage.getItem('visited') !== 'yes') {
```
Comment (Weiqi97) [inline lexos/static/js/scripts_base.js]: Great work! One last thing is that.. Could you please attempt to create a function for this part and call the function in the existing: ```$(function () {})```. Add if the function works, please add a doc string for it.

File: lexos/static/js/scripts_base.js [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+    localStorage.setItem('visited', 'yes')
+    $('#toggler').popover({
+      'html': 'true',
+      'title': 'In the Margins!',
+      'content': 'View instructions for any screen in Lexos by clicking the In the Margins Tab!\n<div class="text-center"><button type="button" id="gotit" class="btn btn-primary" style="background-color: #0068AF">Got it!</button></div>'
+    }).popover('show').data('bs.popover').tip().css({
+      'width': '170px',
+      'text-align': 'center'
+    })
+    $('#gotit').on('click', function () {
```
Comment (Weiqi97) [inline lexos/static/js/scripts_base.js]: I don't really remember if we agreed on using ```.click``` or ```on('click')```..

File: lexos/static/js/scripts_base.js (hunk 1 above)
Comment (danny98m) [reply lexos/static/js/scripts_base.js]: Oh yes my bad I should have done that before after testing but forgot. Just did this.

File: lexos/static/js/scripts_base.js (hunk 2 above)
Comment (danny98m) [reply lexos/static/js/scripts_base.js]: I believe we actually agreed on .click. Just changed it.

**Pull request merged**